api/features/versioning/versioning.py

```diff
@@ -0,0 +1,92 @@
+import typing
```

Comment:
Should this module be renamed to versioning_service?

Reply:
Yeah, that's a fair comment, Have done so.